doc/interfaces-forward.scd

```diff
@@ -0,0 +1,45 @@
+interfaces-forward(5)
+
+# NAME
+
+*interfaces-forward* - forwarding vocabulary for the interfaces(5) file format
+
+# DESCRIPTION
+
+Linux allows for configuration of IP packet forwarding behavior on a protocol
+and interface basis.  The following options allow for this configuration.
+
+# FORWARDING-RELATED OPTIONS
+
```

Comment:
We might want to add 

"If all or some of the following options are not given ifupdown-ng will not touch those settings and the settings configured via sysctl mechanism etc. will be used."